### PR TITLE
Support for Logitech Sight

### PR DESCRIPTION
**Pull request by vcdmp**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by vcdmp** [comment]
Work in progress, going thru integration testing. Will update PR when ready.

File: plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c [hunk 1]
```diff
@@ -1303,6 +1350,123 @@ fu_logitech_bulkcontroller_device_setup(FuDevice *device, GError **error)
...
+					fu_device_set_logical_id(child, name);
+					fu_device_set_version(
+					    child,
+					    json_object_get_string_member(json_device, "sw"));
+					if (json_object_has_member(json_device, "serial"))
+						fu_device_set_serial(
+						    child,
+						    json_object_get_string_member(json_device,
+										  "serial"));
+					if (json_object_has_member(json_device, "type")) {
```
Comment (hughsie) [inline plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c:1442]: this is way too deep; can you abstract it out into a few new functions please.

File: plugins/logitech-bulkcontroller/logitech-bulkcontroller.quirk [hunk 2]
```diff
@@ -12,3 +12,7 @@ Flags = is-mini
 [USB\VID_046D&PID_087C]
 Plugin = logitech_bulkcontroller
 InstallDuration = 1500
+
+[USB\VID_046D&PID_087A]
+Plugin = logitech_bulkcontroller
+InstallDuration = 1500
```
Comment (hughsie) [inline plugins/logitech-bulkcontroller/logitech-bulkcontroller.quirk:18]: needs a newline at the end of the file

**Comment by hughsie** [comment]
Alternate proposal in https://github.com/fwupd/fwupd/pull/7466

**Comment by hughsie** [comment]
Let's continue in https://github.com/fwupd/fwupd/pull/7466

**Pull request closed without merging**